Apps/FCM/ios/Podfile

```diff
@@ -0,0 +1,67 @@
+# Resolve react_native_pods.rb with node to allow for hoisting
+require Pod::Executable.execute_command('node', ['-p',
+  'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
+
+platform :ios, 13.0
+prepare_react_native_project!
+use_modular_headers!
+
```

Comment:
I think it is good to mention as a comment that we are using `use_modular_headers` globally and need not have it added to specific Firebase libraries. In the past, I have hands with FCM not working in the react native project because `:modular_headers => true` and `use_modular_headers` are missing in the podfile.

Reply:
I'm neither much familiar with these options nor really sure if any of these can work for us. I added `use_modular_headers` because this was the only option that made it work for me with FCM. Since you are more familiar with these options, can you please suggest the comment that we should be adding here?

Reply:
Of course. I created this small FCM sample app and [added this code in podfile](https://github.com/customerio/mobile-sample-apps/blob/b2594643c4e31b14381d2093ad9cd3236a29ca34/ReactNative/ObjectiveC-FCM/ios/Podfile#L14-L18 ).

Reply:
I tried updating it as follows

```ruby
  pod 'customerio-reactnative/fcm', :path => '../node_modules/customerio-reactnative'
  pod 'Firebase', :modular_headers => true
  pod 'FirebaseMessaging', :modular_headers => true
  pod 'FirebaseCore', :modular_headers => true
  pod 'GoogleUtilities', :modular_headers => true;
```

But received the following error

```plaintext
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `FirebaseCoreInternal` depends upon `GoogleUtilities-54e75ca4`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
```

Is it because we now moved Firebase dependencies to our iOS SDK?

Not much familiar with these options so I'm open to try anything that works and you think is a better option here 😅